keras/engine/topology.py

```diff
@@ -1621,10 +1630,13 @@ def __init__(self, inputs, outputs, name=None):
                                                    i,
                                                    layer.__class__.__name__))
             self.input_names.append(layer.name)
-            if layer.is_placeholder:
+            if getattr(layer, 'is_placeholder', False) is True:
```

Comment:
Is this preferred over simply saying `getattr(layer, 'is_placeholder', False)`? It can probably be assumed that is_placeholder is a boolean.